src/gui/folder.cpp

```diff
@@ -511,6 +520,13 @@ void Folder::createGuiLog( const QString& filename, LogStatus status, int count,
                 text = tr("%1 has been moved to %2.").arg(file).arg(renameTarget);
             }
             break;
+        case LogStatusConflict:
+            if( count > 1 ) {
+                text = tr("%1 has and %2 other files have sync conflict.").arg(file, count-1);
```

Comment:
I don't thik this prints what you think it prints.  the second parameter to your arg is the fieldWidth.  Also the text reads weird.
I'd write:

tr("%1 and %n other files have sync conflict.", "", count-1).arg(file)